web/src/utils/context/userContext.tsx

```diff
@@ -37,33 +40,36 @@ const UserProvider: React.FC<Props> = ({ children }) => {
 
     // await Auth.currentSession();
 
-    const me = await getMe();
+    const me = await getMe().then((user) => {
```

Comment:
Why the change here?
this effectively means that me is never defined?

Using await + .then doesn't make sense

Reply:
Good catch, was testing a race condition. Reverted it